addons/transifex/models/transifex_code_translation.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import psycopg2
+import werkzeug.urls
+from configparser import ConfigParser
+from os import pardir
+from os.path import isfile, join as opj
+
+import odoo
+from odoo import models, fields, _
+from odoo.tools.translate import CodeTranslations
+
+
+class TransifexCodeTranslation(models.Model):
+    _name = "transifex.code.translation"
+    _description = "Code Translation"
+    _log_access = False
+
+    source = fields.Text(string='Code')
+    value = fields.Text(string='Translation Value')
+    module = fields.Char(help="Module this term belongs to")
+    lang = fields.Selection(selection='_get_languages', string='Language', validate=False)
+    transifex_url = fields.Char("Transifex URL", compute='_compute_transifex_url', store=True, precompute=True,
```

Comment:
Why is this field stored?  Is it necessary?

Reply:
I guess to compute it only once, when you refine your search, etc. To investiguate which one is faster when going through the list of terms